release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -77,6 +77,11 @@ Because {product-title} {product-version} now uses a {op-system-base} 9.2 based
 
 User-defined tags for Microsoft Azure was previously introduced as Technology Preview in {product-title} 4.13 and is now generally available in {product-title} 4.14. For more information, see xref:../installing/installing_azure/installing-azure-customizations.adoc#installing-azure-user-defined-tags_installing-azure-customizations[Configuring the user-defined tags for Azure].
 
+[id="ocp-4-14-user-defined-tags-gcp"]
+==== User-defined tags for Google Cloud Platform (Technology Preview)
```

Comment:
How about "**User-defined labels and user-defined tags for Google Cloud Platform (Technology Preview)**" instead? As the 4.14 epic [CORS-2455 User defined labels for GCP Resources(TP)](https://issues.redhat.com/browse/CORS-2455) introduced both user-defined labels and user-defined tags. Thanks!